contrib/babelfishpg_tsql/src/multidb.c

```diff
@@ -98,6 +98,13 @@ rewrite_object_refs(Node *stmt)
 				switch(cmd->subtype)
 				{
 				case AT_AddColumn:
```

Comment:
Your change affects `AT_AddColumn` (https://www.tutorialspoint.com/cprogramming/switch_statement_in_c.htm).  Is this intentional?